### PR TITLE
feat(app): allow TestTask to accept the list of test-files

### PR DESCRIPTION
**Pull request by caioaamaral**

### CODE REVIEW

File: lib/roby/app/rake.rb [hunk 1]
```diff
@@ -319,6 +324,10 @@ def run_roby_test(*args, report_name: "report")
                         FileUtils.mkdir_p report_dir
                     end
 
+                    args += test_files.flat_map do |file|
```
Comment (doudou) [inline lib/roby/app/rake.rb:327]: test_files is not initialized, i.e. is is nil by default.

Also, `array.flat_map { [value] }` is equivalent to `array.map { value }`

Comment (caioaamaral) [reply lib/roby/app/rake.rb:327]: fixed

Comment (doudou) [reply lib/roby/app/rake.rb:327]: Nothing changed ... pushed the wrong commit ?

Comment (caioaamaral) [reply lib/roby/app/rake.rb]: I pushed in the wrong branch, probably I need a coffee

**Review by doudou — APPROVED** [review]

**Pull request merged**